pkg/urbit/noun/nock.c

```diff
@@ -1769,6 +1927,17 @@ _n_hint_fore(u3_cell hin, u3_noun bus, u3_noun* clu)
       *clu = u3_nul;
     } break;
 
+    case c3__xray : {
+      u3_noun pri, tan;
+      if ( c3y == u3r_cell(*clu, &pri, &tan) ) {
+        c3_l pri_l = c3y == u3a_is_cat(pri) ? pri : 0;
+        u3t_slog_cap(pri_l, u3i_string("bytecode of"), u3k(tan));
+        _cn_xray(pri_l, fol);
+      }
```

Comment:
The output here isn't quite right; `tan` should be the caption of the bytecode, not it's own output with a constant caption (`"bytecode of"`). Given the changes described above, this should call `u3t_slog_cap(pri_l, u3k(tan), _cn_etch_bytecode())`.

Reply:
I've fixed the code per your notes.
I've tested it and it seems to be working correctly.